Dockerfile.govuk-base

```diff
@@ -6,7 +6,7 @@
 FROM buildpack-deps
 
 # Install chrome and its dependencies
-RUN apt-get update -qq && apt-get install -y libxss1 libappindicator1 libindicator7
+RUN apt-get update -qq && apt-get install -y libxss1 libappindicator1 libindicator7 --no-install-recommends
```

Comment:
I know Docker favours using difficult and unmaintainable Dockerfiles, but if you wanted to avoid passing `--no-install-recommends` each time you invoke apt, I believe it's possible to effect the same change by amending the apt configuration.